uiowa/uiowa
# Classroom Schedule for Room Block

Resolves #6045 

# How to test
- Code review
```
ddev blt ds --site=classrooms.uiowa.edu && ddev drush @classrooms.local uli node/add/room
```
- Use Art building west 240 as test and test against https://api.maui.uiowa.edu/maui/api/pub/registrar/courses/AstraRoomSchedule/2023-01-25/2023-01-25/abw/240 (update date as needed)
- Verify that on save room schedule displays
- Verify that Technology guide link is pre-populated and renders on save
- Check out https://classrooms.dev.drupal.uiowa.edu/art-building-west-240-0 on dev and see if it is cached
